aiven/client/cli.py

```diff
@@ -4414,14 +4385,7 @@ def project__list(self) -> None:
     @arg.project
     @arg("--name", help="New project name")
     @arg("--account-id", help="Account ID of the project")
```

Comment:
this one should probably be migrated to  be `@arg.parent_id` (and accompanying lookup?)